packages/web/src/io/generateOutputFileFileSystem.js

```diff
@@ -1,7 +1,7 @@
 const FileSystemApiErrorHandler = require('./utils')
 
 module.exports = function generateOutputFileFileSystem (extension, blob, callback) {
-  var request = window.requestFileSystem || window.webkitRequestFileSystem
+  var request = window.requestFileSystem || window.webkitStorageInfo.requestQuota
```

Comment:
Should probably have a path check on window.webkitStorageInfo to avoid unhandled exception